### PR TITLE
Adding Bottom Navigation to Mobile / Small widths

### PR DESCRIPTION
**Pull request by DacoGT**
Add in BottomNavigation to support mobile platforms.
Use useMediaQuery and breakpoints to determine screen width.

### CODE REVIEW

**Comment by LarsKemmann** [comment]
Very cool! It'll be really nice to have this in place. It looks like this is still a draft pull request - make sure to mark it as ready for review when you're at that point.

File: src/caretogether-pwa/src/App.tsx [hunk 1]
```diff
@@ -187,7 +196,7 @@ function App() {
     <div className={classes.root}>
       <CssBaseline />
       <Router>
-        <AppBar position="absolute" className={clsx(classes.appBar, open && classes.appBarShift)}>
+        <AppBar position="absolute" className={clsx(classes.appBar, (open && !isMobile) && classes.appBarShift)}>
           <Toolbar className={classes.toolbar}>
```
Comment (LarsKemmann) [inline src/caretogether-pwa/src/App.tsx:200]: Just a note that there's a merge conflict on this line, make sure you merge `main` into your branch to resolve the conflict.

File: src/caretogether-pwa/src/App.tsx [hunk 2]
```diff
@@ -179,6 +184,10 @@ function App() {
   const handleDrawerClose = () => {
     setOpen(false);
   };
+  const [value, setValue] = React.useState(0);
```
Comment (LarsKemmann) [inline src/caretogether-pwa/src/App.tsx:187]: I don't think you're actually using this (see my note below), you can remove this.

File: src/caretogether-pwa/src/App.tsx [hunk 3]
```diff
@@ -221,61 +230,74 @@ function App() {
...
+                <VolunteerProgress />
+              </Route>
+              <Route path="/contacts">
+                <Contacts />
+              </Route>
+              <Route path="/communities">
+                <Communities />
+              </Route>
+              <Route>
+                <Redirect to="/arrangements" />
```
Comment (LarsKemmann) [inline src/caretogether-pwa/src/App.tsx:284]: Similar note about a merge conflict on this line.

File: src/caretogether-pwa/src/App.tsx [hunk 4]
```diff
@@ -221,61 +230,74 @@ function App() {
...
+              </Route>
+            </Switch>
+          </React.Suspense>
           {/* </Container> */}
+          {isMobile ? <BottomNavigation
+            value={value}
+            onChange={(event, newValue) => {
+              setValue(newValue);
+            }}
```
Comment (LarsKemmann) [inline src/caretogether-pwa/src/App.tsx:293]: This (`value` and `onChange`) doesn't seem to do anything, can you remove it?

**Review by LarsKemmann — CHANGES_REQUESTED** [review]
@DacoGT Very nice work!! It looks fantastic. 😀 Take a look at the comments I added, it's all minor cleanup stuff (I think). Otherwise this is good to go!

File: src/caretogether-pwa/src/App.tsx (hunk 2 above)
Comment (DacoGT) [reply src/caretogether-pwa/src/App.tsx:187]: The value on the BottomNavigation component uses these to set the classes to highlight the correct button. I did however find a bug. If you from full to small size and the BottomNavigation pops up, the highlighted state isn't sync'd correctly. I'll see if I can fix that quick

File: src/caretogether-pwa/src/App.tsx (hunk 1 above)
Comment (DacoGT) [reply src/caretogether-pwa/src/App.tsx:200]: I'll be sure to merge, sort out the conflicts then make this a regular Pull Request

**Comment by DacoGT** [comment]
Looks like my merge made this look awful :(

**Comment by DacoGT** [comment]
> Looks like my merge made this look awful :(
I'm having trouble fixing this branch. I created a new one and started a new pull request
@LarsKemmann

**Comment by LarsKemmann** [comment]
Closing this since changes were merged via the other two PRs.

**Pull request closed without merging**